modules/ssh/manifests/userconfig.pp

```diff
@@ -0,0 +1,122 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This entire file is a whole lot of cruft.  It is the type stuff don't want to bring over from puppetagin.  And besides, user keys are already handled in the 'user' module.

https://github.com/mozilla-platform-ops/ronin_puppet/blob/master/modules/users/manifests/user_ssh_config.pp